axelar-chains-config/src/utils/verifyContract.js

```diff
@@ -27,9 +27,26 @@ const verifyContract = (env, chain, contract, args, options = {}) => {
     console.log(`Verifying contract ${contract} with args '${stringArgs.join(',')}'`);
     console.log(cmd);
 
-    execSync(cmd, { stdio: 'inherit' });
+    await new Promise((resolve, reject) => {
+        exec(cmd, { stdio: 'inherit' }, (error, stdout, stderr) => {
+            console.log(stdout);
 
-    console.log('Verified!');
+            if (error) {
+                if (stderr && stderr.includes('Reason: Already Verified')) {
```

Comment:
the reason i usually get is this. e.g. when verifying gas service v6.3.1 upgrade on testnet
`Reason: Fail - Unable to verify. Compiled contract deployment bytecode does NOT match the transaction deployment bytecode.`

can you test it on that?

Reply:
ok, that seems to be the bigger issue then on fantom. Have you tried investigating it? Do you just verify it with the flattened contract in that case? Maybe reach out to etherscan team about this issue?